Specs/FMDBSwift/0.0.2/FMDBSwift.podspec.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "FMDBSwift",
+  "version": "0.0.2",
+  "license": "Apache License, Version 2.0",
+  "summary": "Modified FMDB version to remove non modular headers (Swift Compatible)",
+  "homepage": "",
+  "authors": {
+    "Rafael Ring": "rafael.ring@taqtile.com"
+  },
+  "platforms": {
+    "ios": null
```

Comment:
Well, I don't know... 

This is a temp spec for a fork to make it work with swift and the [original spec](https://github.com/robertmryan/fmdb/blob/master/FMDB.podspec) does not say anything and it worked with all version I tested that supported swift so far...